manual/migrations/migration-cli.md

```diff
@@ -1 +1,53 @@
 # Migration CLI
+
+
+## Available Commands
+
+There are a few commands that makes it a more enjoyable experience to your everyday usage of Clear ORM.
+
+
+### Generators
+
+The model and scaffold generators will create migrations appropriate for adding a new model. Clear ORM provides a consice DSL for defining migrations, but these can also be generated via the CLI. 
+
+```sh
+Usage:
+  /home/eperez/snap/crystal/common/.cache/crystal/crystal-run-cli.tmp generate [flags...] [arg...]
+
+Generate code automatically
+
+Flags:
+  --help         # Displays help for the current command.
+  --no-color     # Cancel color output
+  --verbose, -v  # Display verbose informations during execution
+
+Subcommands:
+  migration      # Generate a new migration
+  model          # Create a new model and the first migration
+  new:kemal      # Create a new project with Kemal
+```
+
+### Migration 
+
+Migrations are a convenient way to alter the database schema over time in a consistent and easy way. Clear ORM provides a beautiful DSL so that you don't have to write SQL by hand, allowing your schema and changes to be database independent.
+
+```sh
+Usage:
+  /home/eperez/snap/crystal/common/.cache/crystal/crystal-run-cli.tmp migrate [flags...] [arg...]
```

Comment:
The binary name here gives clue on your system architecture and could confuse the reader. I would recommend you to push a fix with `clear-cli` or other binary name instead.